governance.md

```diff
@@ -288,39 +344,36 @@ raising money from private foundations and donors or a for-profit company
 building proprietary products and services that leverage Project Software and
 Services. Funding acquired by Institutional Partners to work on The Project is
 called Institutional Funding. However, no funding obtained by an Institutional
-Partner can override The Project BDFL and Steering Council. If a Partner has
+Partner can override The Project LD and Steering Council. If a Partner has
 funding to do Matplotlib work and the Council decides to not pursue that
 work as a project, the Partner is free to pursue it on their own. However in
 this situation, that part of the Partner’s work will not be under the
 Matplotlib umbrella and cannot use the Project trademarks in a way that
 suggests a formal relationship.
 
-To acknowledge institutional contributions, Institutional Partners
-receive the following benefits:
+To acknowledge institutional contributions, Institutional Partners receive the
+following benefits:
 
 -   Acknowledged on the Matplotlib websites, in talks and T-shirts.
--   Ability to acknowledge their own funding sources on the Matplotlib
-    websites, in talks and T-shirts.
--   Ability to influence the project through the participation of their Council
-    Member.
+-   Ability to acknowledge their own funding sources on the Matplotlib websites,
+    in talks and T-shirts.
+
 
 Changing the Governance Documents
 =================================
 
-Changes to the governance documents are submitted via a GitHub pull
-request to The Project's governance documents GitHub repository at
+Changes to the governance documents are submitted via a GitHub pull request to
+The Project's governance documents GitHub repository at
```

Comment:
is this gonna change if this all moves to the main docs?

Reply:
no, I think it should stay in it's own repository, even if we publish build artifacts as part of the main docs.